src/components/layout/Duo/Duo.jsx

```diff
@@ -0,0 +1,23 @@
+import PropTypes from 'prop-types'
+import React from 'react'
+
+const Duo = ({ isDisabled, label }) =>
+  !isDisabled && (
+    <div className="duo">
+      <span>{'DUO'}</span>
```

Comment:
```suggestion
      <span>{'duo'}</span>
```
Et met le majuscule via le CSS.